scripts/pipelines/azure-devops/templates/build/build-pipeline.cfg

```
@@ -17,4 +17,22 @@ scriptFilePath=".pipelines/scripts"
 function copyScript {
     # Copy the script.
     cp "${hangarPath}/${templatesPath}/${language}-${scriptFile}" "${localDirectory}/${scriptFilePath}/${scriptFile}"
-}
\ No newline at end of file
+}
+
+# Function that adds the variables to be used in the pipeline.
+function addPipelineVariables {
+    # if the user did not specify a custom target-directory 
+    # we default to the language specific defaults e.g. target for java, ./ for node
+
+    if test -z $targetDirectory
+    then
+        case $language in 
+            node) targetDirectory="./" ;;
+            quarkus) targetDirectory="./target/" ;;
+            quarkus-jvm) targetDirectory="./target/";;
```

```suggestion
            quarkus*) targetDirectory="./target/" ;;
```

Would this work? Try before accepting the suggestion.